thoth/lab/inspection.py

```diff
@@ -709,15 +745,13 @@ def plot_interpolated_statistics_of_inspection_parameters(
     plt.show()
 
 
-def create_inspections_time_dataframe(
-    df_inspection_batches_dict: dict, inspection_identifiers: List[str], n_parallel: int = 6
-) -> pd.DataFrame():
+def create_inspection_time_dataframe(df_inspection_batches_dict: dict, n_parallel: int = 6) -> pd.DataFrame():
     """Create pd.Dataframe of time of inspections for build and job."""
     tot_time_builds = []
     tot_time_jobs = []
     tot_time_sum_builds_and_jobs = []
 
-    for identifier, dataframe in df_inspection_batches_dict.items():
+    for identifier, dataframe in list(df_inspection_batches_dict.items()):
```

Comment:
is the `list(...)` necessary here?

Reply:
I have to correct it, thanks Marek :beers: